internal/provider/incident_workflow_resource.go

```diff
@@ -51,6 +59,24 @@ func (r *IncidentWorkflowResource) Schema(ctx context.Context, req resource.Sche
 		},
 	}
 
+	paramBindingsAttribute := schema.ListNestedAttribute{
+		Required: true,
+		NestedObject: schema.NestedAttributeObject{
+			Attributes: map[string]schema.Attribute{
+				"array_value": schema.SetNestedAttribute{
+					Optional: true,
+					NestedObject: schema.NestedAttributeObject{
+						Attributes: paramBindingValueAttributes,
+					},
+				},
+				"value": schema.SingleNestedAttribute{
+					Optional:   true,
+					Attributes: paramBindingValueAttributes,
+				},
+			},
+		},
+	}
```

Comment:
feels good to flatten this nest

Reply:
Can you explain what you mean here? i'm not quite sure!

Reply:
it was just very deeply nested >>>>>